mkfs/common.c

```diff
@@ -127,11 +127,14 @@ static int btrfs_create_tree_root(int fd, struct btrfs_mkfs_config *cfg,
 			memcpy(root_item.uuid, uuid, BTRFS_UUID_SIZE);
 			btrfs_set_stack_timespec_sec(&root_item.otime, now);
 			btrfs_set_stack_timespec_sec(&root_item.ctime, now);
+			btrfs_set_stack_inode_flags(inode_item,
```

Comment:
Please keep it on one line, in progs it's not that strict like in kernel code, so overflow 80+ is OK.